test/test_helper.exs

```diff
@@ -1 +1 @@
-ExUnit.start()
+ExUnit.start(timeout: :infinity)
```

Comment:
what's the impact of this on session

Reply:
@ankur1708 this is a configuration for breakpoints. When we want to place an IEx.pry in our code to pause execution, it will time out after only a short amount of time. This change allows you to stay at the breakpoint indefinitely, and does not impact sessions